prog/mutation.go

```diff
@@ -96,7 +96,13 @@ func (p *Prog) Mutate(rs rand.Source, ncalls int, ct *ChoiceTable, corpus []*Pro
 								arg.Data = mutateData(r, data, minLen, maxLen)
 							case sys.BufferString:
 								if r.bin() {
-									arg.Data = mutateData(r, append([]byte{}, arg.Data...), int(0), ^int(0))
+									minLen := int(0)
+									maxLen := ^int(0)
```

Comment:
`^int(0)` gives `-1` because it is signed type

there is a bunch of similar places in this file, please fix them